File: app/src/main/java/com/snowdango/numac/NumAcMain/ClickTextChanger.java
package com.snowdango.numac.NumAcMain;

import android.util.Log;
import android.widget.TextView;

import com.snowdango.numac.R;


public class ClickTextChanger{

    public void clickEvents(int button, Boolean b, TextView textView){

        if(b) {
            switch (button) {
                case R.id.button1:
                    textView.setText(textView.getText() + "1");
                    Log.d("E","1");
                    break;
                case R.id.button2:
                    textView.setText(textView.getText() + "2");
                    Log.d("E","2");
                    break;
                case R.id.button3:
                    textView.setText(textView.getText() + "3");
                    Log.d("E","3");
                    break;
                case R.id.button4:
                    textView.setText(textView.getText() + "4");
                    Log.d("E","4");
                    break;
                case R.id.button5:
                    textView.setText(textView.getText() + "5");
                    Log.d("E","5");
                    break;
                case R.id.button6:
                    textView.setText(textView.getText() + "6");
                    Log.d("E","6");
                    break;
                    case R.id.button7:
                    textView.setText(textView.getText() + "7");
                        Log.d("E","7");
                    break;
                case R.id.button8:
                    textView.setText(textView.getText() + "8");
                    Log.d("E","8");
                    break;
                case R.id.button9:
                    textView.setText(textView.getText() + "9");
                    Log.d("E","9");
                    break;
                case R.id.button10:
                    textView.setText("");
                    break;
                case R.id.button0:
                    textView.setText(textView.getText() + "0");
                    Log.d("E","0");
                    break;
                case R.id.button11:
                    textView.setText(textView.getText() + "#");
                    Log.d("E","#");
                    break;

            }
        } // button switch



    }

}

File: app/src/main/java/com/snowdango/numac/controller/ClickTextChanger.kt
package com.snowdango.numac.controller

import android.util.Log
import android.widget.TextView
import com.snowdango.numac.R

/*
Main Activity pushed button ,listen this class.
this class can textView's text.
 */

class ClickTextChanger {
    fun clickEvents(button: Int, b: Boolean, textView: TextView) {
        if (b) {
            when (button) {
                R.id.button1 -> {
                    textView.text = textView.text.toString() + "1"
                    Log.d("E", "1")
                }
                R.id.button2 -> {
                    textView.text = textView.text.toString() + "2"
                    Log.d("E", "2")
                }
                R.id.button3 -> {
                    textView.text = textView.text.toString() + "3"
                    Log.d("E", "3")
                }
                R.id.button4 -> {
                    textView.text = textView.text.toString() + "4"
                    Log.d("E", "4")
                }
                R.id.button5 -> {
                    textView.text = textView.text.toString() + "5"
                    Log.d("E", "5")
                }
                R.id.button6 -> {
                    textView.text = textView.text.toString() + "6"
                    Log.d("E", "6")
                }
                R.id.button7 -> {
                    textView.text = textView.text.toString() + "7"
                    Log.d("E", "7")
                }
                R.id.button8 -> {
                    textView.text = textView.text.toString() + "8"
                    Log.d("E", "8")
                }
                R.id.button9 -> {
                    textView.text = textView.text.toString() + "9"
                    Log.d("E", "9")
                }
                R.id.button10 -> textView.text = ""
                R.id.button0 -> {
                    textView.text = textView.text.toString() + "0"
                    Log.d("E", "0")
                }
                R.id.button11 -> {
                    textView.text = textView.text.toString() + "#"
                    Log.d("E", "#")
                }
            }
        } // button switch
    }
}